--- a/CMakeLists.txt	2025-11-07 09:21:33.985922903 +0100
+++ b/CMakeLists.txt	2025-11-07 09:22:20.621132722 +0100
@@ -26,7 +26,6 @@
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/build/cmake/modules) # Modules from MEGAcmd
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/sdk/cmake/modules) # Modules from MEGAsdk
 
-set(VCPKG_ROOT "${CMAKE_CURRENT_LIST_DIR}/../vcpkg" CACHE PATH "If set, it will build and use the VCPKG packages defined in the manifest file")
 
 include(detect_host_architecture)
 
